### ISSUE TITLE
Keyboard help dialog strings should be translatable

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/gravity-force-lab-basics#56]
Even though they are a11y strings, they strings in the keyboard help dialog are visible so they need to be translatable. @mbarlow12 can you please do this?

For #49.

**Comment by jessegreenberg** [comment]
![capture](https://user-images.githubusercontent.com/6396244/36928066-74efcd54-1e3f-11e8-9ac5-c8340447ea04.PNG)

**Comment by mbarlow12** [comment]
Is it possible that this issue is related to https://github.com/phetsims/a11y-research/issues/65?  In [SliderControlsHelpContent.js](https://github.com/phetsims/scenery-phet/blob/666a80bbff3d4ddf6060e33e57e2c90f60a18faf/js/keyboard/help/SliderControlsHelpContent.js), we're including a11y strings in the same fashion as normal strings.

I'm not really sure how to proceed on this one. Maybe table for dev-a11y discussion tomorrow (Wed) @jessegreenberg ?

**Comment by jessegreenberg** [comment]
We want to keep visible strings are translatable, but keep invisible strings that are purely there for screen reader access non-translatable. [SliderControlsHelpContent](https://github.com/phetsims/scenery-phet/blob/666a80bbff3d4ddf6060e33e57e2c90f60a18faf/js/keyboard/help/SliderControlsHelpContent.js) adheres to this. The dialog in GFL:B doesn't have screen reader text yet, but the visual strings should still be translatable.
